website/content/docs/job-specification/consul.mdx

```diff
@@ -99,9 +100,7 @@ The [`template`][template] block can use the Consul token as well.
 - `partition` `(string: "")` - When this field is set, a constraint will be
   added to the group or task to ensure that the allocation is placed on a Nomad
   client that has a Consul Enterprise agent in the specified Consul [admin
-  partition][]. Note that Consul Community Edition agents are not assigned to
-  any admin partition, so this field should not be used without Consul
-  Enterprise.
+  partition][].
```

Comment:
As I noted before, the admin partition is _cannot be fingerprinted_ when using Consul CE, so we should leave this note here. If you want to change the wording to specifically talk about fingerprinting that's fine, but if you set this field to `"default"` with Consul CE, none of your workloads will be placed!